index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Project</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+   <section class="body">
+      <header>
+         <nav class="welcome">Hi there</nav>
```

Comment:
- [ ] Please kindly wrap your logo with a tag as suggested earlier by the previous reviewer for good user experience. 👍 
```suggestion
         <a class="welcome">Hi there</a>
```